evm_arithmetization/src/all_stark.rs

```diff
@@ -419,3 +424,17 @@ fn ctl_poseidon_general_output<F: Field>() -> CrossTableLookup<F> {
         poseidon_stark::ctl_looked_general_output(),
     )
 }
+
+#[cfg(test)]
+mod tests {
+    use plonky2::field::goldilocks_field::GoldilocksField;
+
+    use super::*;
+
+    type F = GoldilocksField;
+
+    #[test]
+    fn check_num_ctls() {
+        assert_eq!(all_cross_table_lookups::<F>().len(), NUM_CTLS);
+    }
```

Comment:
I can't add such check for the `MEMORY_CTL_IDX` constant, because the `looked_table` field within `CrossTableLookup` is crate-private, though I could change it in the sibling PR if wanted.

Reply:
I think we can add a comment before `ctl_memory(),` to indicate we cannot change its position.